lib/rules/no-restricted-globals.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
                         type: "object",
                         properties: {
                             name: { type: "string" },
-                            message: { type: "string" }
+                            message: { type: "string" },
+                            isFullMessage: { type: "boolean" }
```

Comment:
This naming doesn't make much sense to me. Maybe `replaceDefaultMessage` is clearer?

Reply:
Option renamed to replaceDefaultMessage.